tests/integration/api_v2_product_write.t

```diff
@@ -103,6 +103,29 @@ my $tests_ref = [
 		path => '/api/v2/product/1234567890003',
 		expected_status_code => 404,
 	},
+	# Test that we use the language of the interface (lc) for language fields without a language suffix
+	{
+		test_case => 'post-product-ingredients-text-without-language',
+		method => 'POST',
+		path => '/cgi/product_jqm_multilingual.pl',
+		form => {
+			cc => "uk",
+			lc => "en",
+			lang => "fr",
```

Comment:
A comment here to explain this is welcome: what is the difference between lc and cc (I'm always lost)